### PR TITLE
Modify `.card-header-tabs .nav-link.active` colors

### LINKED ISSUE #28820: "Card Integrated Tabs" active background should default to $card-bg.
### Is your feature request related to a problem? Please describe...

When using [Card Integrated Tabs](https://getbootstrap.com/docs/4.1/components/card/#navigation), the active tab background defaults to `$body-bg`.

### Describe the solution you'd like

The active tab background should default to `$card-bg` rather than `$body-bg`, to match the background of the card.

### Describe alternatives you've considered

It can easily be overridden with scss, e.g.
```scss
.card-header-tabs .nav-link.active {
  background: $card-bg;
  border-color: $card-border-color $card-border-color $card-bg;
}
```
 but I believe it makes sense to not have to go down that road.

### Additional context

Screen Shot 2019-05-23 at 4 31 17 PM

### PR DESCRIPTION
**Pull request by shaneparsons**
Set `.card-header-tabs .nav-link.active` background / border colors to better match card.

Fixes #28820.

### CODE REVIEW

**Comment by shaneparsons** [comment]
For #28820.

**Comment by ysds** [comment]
Should also care for hover?

**Comment by shaneparsons** [comment]
Since `background-color` isn't actually changed on hover, and the `border-color` changes are fine, I don't think any hover styles are needed.

I did make a change to my pull request though:

Previous:
`border-color: $card-border-color $card-border-color $card-bg;`

New:
`border-bottom-color: $card-bg;`

Now only the bottom border colour changes, to match the card background.

**Comment by shaneparsons** [comment]
Here are some screenshots of the difference the pull request makes:

Before:
Screen Shot 2019-05-27 at 3 21 46 PM

After:
Screen Shot 2019-05-27 at 3 22 37 PM

Note: the first tab is active, the second tab is hovered.

File: scss/_card.scss [hunk 1]
```diff
@@ -104,6 +104,11 @@
   margin-bottom: -$card-spacer-y;
   margin-left: -$card-spacer-x / 2;
   border-bottom: 0;
+
+  .nav-link.active {
+    background: $card-bg;
```
Comment (mdo) [inline scss/_card.scss]: Don’t use the shorthand, use `background-color` so it doesn’t override additional `background-` properties.

```suggestion
    background-color: $card-bg;
```

**Review by mdo — COMMENTED** [review]
I think I’m in favor of this. Feels like a helpful consideration for folks who may modify the `$body-bg` to be something different. Otherwise though, it’s unnecessary. Thoughts from @twbs/css-review for how to account for that and avoid including CSS that’s not used by default?

**Comment by shaneparsons** [comment]
@mdo Indeed. Some frameworks like Laravel modify the `$body-bg` by default too, so it's one less thing that developers have to worry about.

**Comment by shaneparsons** [comment]
There's probably a way to modify this to play nice with [#background-and-color](https://getbootstrap.com/docs/4.3/components/card/#background-and-color) as well, but that's a seperate issue.

**Review by ysds — CHANGES_REQUESTED** [review]
> Thoughts from @twbs/css-review for how to account for that and avoid including CSS that’s not used by default?

@shaneparsons Could you change as the follows? 😄

```scss
  .nav-link.active {
    background-color: $card-bg;
    border-bottom-color: $card-bg;
  }
```

to

```scss
  @if $nav-tabs-link-active-bg != $card-bg {
    .nav-link.active {
      background-color: $card-bg;
      border-bottom-color: $card-bg;
    }
  }
```

**Comment by shaneparsons** [comment]
> > Thoughts from @twbs/css-review for how to account for that and avoid including CSS that’s not used by default?
> 
> @shaneparsons Could you change as the follows? 😄
> 
> ```scss
>   .nav-link.active {
>     background-color: $card-bg;
>     border-bottom-color: $card-bg;
>   }
> ```
> 
> to
> 
> ```scss
>   @if $nav-tabs-link-active-bg != $card-bg {
>     .nav-link.active {
>       background-color: $card-bg;
>       border-bottom-color: $card-bg;
>     }
>   }
> ```

Done!

See my edit to https://github.com/twbs/bootstrap/pull/28833#issuecomment-496524571 and let me know if the additions are of any interest to you.

**Comment by ysds** [comment]
I think it's over-engineering to integrate with the background utilities. It would be better to solve it with custom variables in the future (v6?)

**Review by ysds — APPROVED** [review]

**Comment by shaneparsons** [comment]
> I think it's over-engineering to integrate with the background utilities. It would be better to solve it with custom variables in the future (v6?)

Yeah I agree on that one. I don't have a ton of experience with mixins / utilities, so that was just my hacky take on the solution. I more so just intended to get the conversation started.

**Review by MartijnCuppens — APPROVED** [review]

**Review by mdo — APPROVED** [review]
Thanks, everyone! <3

**Comment by XhmikosR** [comment]
I assume we don't want this backported, right?

**Pull request merged**